.github/workflows/release-apt-get.yml

```diff
@@ -0,0 +1,106 @@
+name: "release-apt-get"
+on:
+  release:
+    types: [released]
+
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release Tag'
+        required: true
+        default: 'latest'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - uses: actions/checkout@v2
+
+      - uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: 'Download Repo Client'
+        env:
+          AZ_SUB: ${{ secrets.AZURE_SUBSCRIPTION }}
+        run: |
+          az storage blob download --subscription  "$AZ_SUB" --account-name gitcitoolstore -c tools -n azure-repoapi-client_2.0.1_amd64.deb -f repoclient.deb --auth-mode login
+
+      - name: "Install Repo Client"
+        env:
+          AZURE_AAD_ID: ${{ secrets.AZURE_AAD_ID }}
+          AAD_CLIENT_SECRET: ${{ secrets.AAD_CLIENT_SECRET }}
+        run: |
+          sudo apt-get install python3-adal --yes
+          sudo dpkg -i repoclient.deb
+          python .github/configure_repoclient.py
```

Comment:
This is now done below, in the `github-script` step, right?

Reply:
I was able to remove the reference to the old script (thanks for calling that out!). However, we still need to install `pythin3-adal`, as it's a repoclient dependency.

Reply:
Okay, thanks for explaining!